exercises/Makefile

```diff
@@ -0,0 +1,140 @@
+# Colors
+COLOR=\033[0;32m
+NC=\033[0m
+BOLD=\033[1m
+
+# Vars
+DOCKER_COMPOSE_CMD  = docker-compose -f docker-compose.yml
+KUBECTL_CONFIG_PATH = .output/kubeconfig.yaml
+KUBECTL_CMD         = kubectl --kubeconfig=${KUBECTL_CONFIG_PATH}
+
+##
+## Targets to manage the local k3s cluster
+## ---------------------------------------
+##
+
+all: help
+help: ## Display this message
+	@grep -E '(^[a-zA-Z0-9_.-]+:.*?##.*$$)|(^##)' Makefile | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m##/[33m/'
+
+start: ## Start k3s cluster
+	@${DOCKER_COMPOSE_CMD} up -d
+	@${DOCKER_COMPOSE_CMD} ps
+
+stop: ## Stop k3s cluster (also purge docker volumes)
+	@${DOCKER_COMPOSE_CMD} down --volumes
+
+cluster-info: ## Get cluster infos
+	${KUBECTL_CMD} cluster-info
+
+# Level start template
+template.level%.start: template.level%.stop
+	@echo "${COLOR}# ---------------------------${NC}"
+	@echo "${COLOR}# Start level$* resources ...${NC}"
+	@echo "${COLOR}# ---------------------------${NC}"
+	@echo "${COLOR}${BOLD}Take a look at 'exercises/level$*/README.md' !${NC}"
+	${KUBECTL_CMD} -n level$* apply -f level$*/.backstage/level$*.yml
+
+# Level stop template
+template.level%.stop:
+	@echo "${COLOR}# -----------------------------${NC}"
+	@echo "${COLOR}# Delete level$* ressources ...${NC}"
+	@echo "${COLOR}# -----------------------------${NC}"
+	-${KUBECTL_CMD} delete namespace level$*
+
+# Level logs template
+template.level%.logs:
+	@echo "${COLOR}# ------------------------------${NC}"
+	@echo "${COLOR}# Display level$* ressources ...${NC}"
+	@echo "${COLOR}# ------------------------------${NC}"
+	-${KUBECTL_CMD} -n level$* get configmap,secret,ingress,svc,deploy,pod,rs,job,cronjob,pv,pvc
+
+##
+## Level 001
+## ---------
+##
+
+level1.start: template.level1.start ## Start level1
```

Comment:
I'm confused.

These are not documented anywhere, and yet I already have levels 1 & 2 on my cluster without doing anything.

Reply:
because of this: https://github.com/hugoShaka/k8s-workshop/blob/master/exercises/docker-compose.yml#L20

Reply:
Forget the part about levels 1 & 2 starting without doing anything, I understand now that you remove it in this PR.

It's OK for me, I'd just like a few lines in the README explaining how to start levels. (non-blocking, can be addressed later, but we should add at least a TODO comment in this case)